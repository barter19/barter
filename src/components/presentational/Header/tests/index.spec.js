import { shallow } from '@vue/test-utils';
import HeaderComponent from '../index';

const renderer = require('vue-server-renderer').createRenderer();

describe('Header Component', () => {
  HeaderComponent.i18n.locale = 'en';

  const ClonedComponent = shallow(HeaderComponent);
  const { vm } = ClonedComponent;

  it('renders to a nice snapshot', () => {
    renderer.renderToString(vm, (err, str) => {
      expect(str).toMatchSnapshot();
    });
  });
});
