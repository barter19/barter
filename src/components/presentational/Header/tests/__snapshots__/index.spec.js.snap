// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders to a nice snapshot 1`] = `
<header class="header">
    <h1>VueCharged Template</h1>
</header>
`;
